log/log_test.go

```diff
@@ -40,7 +41,7 @@ func TestInfoWithFields(t *testing.T) {
 		assert.Equal(t, fields["msg"], "test")
 		assert.Equal(t, fields["level"], "info")
 		assert.Equal(t, fields["key"], "value")
-		assert.Equal(t, fields["pkg"], "log")
+		assert.Equal(t, fields["pkg"], "log.TestInfoWithFields")
```

Comment:
I'm just curious. Will the other packages properly be logged as just the package name?